site/_wiki/Documentation/ConfigurationGuide.md

```diff
@@ -0,0 +1,418 @@
+
+##### `OutOfRangeReport`
+
+Fairly selfdescript, it doesn't actually report any specific values to the
+OS but rather 'resets' the pen out.
+
+##### `XP_PenAuxReport`:
+
+Reports `IAuxReport`, an aux-buttons-only report, ie. no position coordinates.
+
+##### `XP_PenTabletReport`
+
+Reports the following reports:
+
+- `ITabletReport`
+  - The pen's pressure
+  - Pen buttons
+  - It is an `IAbsolutePositionReport` that also reports the following:
+    - The 2-point vector of the pen
+- `ITiltReport`
+  - The 2-point vector of the pen's tilt
+- `IEraserReport`
+  - Whether the eraser is active or not
+
+##### `XP_PenTabletOverflowReport`
+
+An almost identical version of the above `XP_PenTabletReport` except with
+larger ranges for the position coordinate and the data structure adjusted
+accordingly.
+
+##### `TabletReport`
+
+A barebones tablet report that only reports an `ITabletReport`
+
+### Init strings (Feature/OutputInitReport, InitializationStrings)
+
+These are almost always necessary to use to bring the tablet out of HID mode
+and into a more preferred mode, e.g. vendor mode.
+
+Vendor mode is the mode used by the official drivers, and allows
+OpenTabletDriver to support all of the tablets features, assuming the parser
+supports it.
+
+Getting these can be achieved in 2 ways:
+
+#### Guessing
+
+Copying the appropriate init strings from a sibling configuration is usually
+adequate.
+
+#### USB Packet capture
+
+This is usually done with Wireshark (and winpcap). A guide on this is planned
+but not in progress. See [opentabletdriver.github.io#171] for more information.
+
+[opentabletdriver.github.io#171]: https://github.com/OpenTabletDriver/opentabletdriver.github.io/issues/171
+
+### Device strings
+
+In some cases, vendors (especially Huion) might release a tablet with different
+specifications that otherwise has the same ID's as another released tablet.
+
+While this complicates things, we have another tool in our arsenal called
+device strings descriptors.
+
+The configuration format is a dictionary where the key matches the descriptor
+ID (up to 255) and the value is a regex that the string descriptor's value must
+match with.
+
+## Validating values
+
+Assuming you now have a configuration that correctly identifies your tablet,
+you now have to validate that you're using the correct parser and then the
+correct configuration values.
+
+### Validating the parser
+
+This section is **not** for making sure the specifications in the configuration
+is configured correctly, only the parser.
+
+The values below can be validated from entirely within the tablet debugger.
+
+The tablet debugger will show the values as parsed directly from the tablet,
+before any configuration adjustments (such as button count) take place.
+
+If any of the values seems absent or malfunctioning, you are probably
+using the wrong parser.
```

Comment:
Should we mention some symptoms of using the wrong parser?
```suggestion
If any of the values seems absent or malfunctioning (overflowing, abrupt jumps), 
you are probably using the wrong parser.
```
Also crashing is a symptom too, but this could also be a symptom of using the wrong interface and OTD trying to parse garbage makes it crash.

In cases where you are trying to write/figure out what parser to use and you keep getting crashes, recommending the `PassthroughReportParser` could be good.